.travis.yml

```diff
@@ -5,12 +5,7 @@ language: java
 jdk:
   - oraclejdk8
 
-addons:
-  apt:
-    packages:
-      - oracle-java8-installer
-      - oracle-java8-set-default
+script: ./gradlew clean build
 
-install: true # skip default `mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V` call
-
-script: mvn clean verify
\ No newline at end of file
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Its for pinging the code coverage tool to fetch the Jacoco code coverage reports: https://codecov.io/gh/commercetools/commercetools-sync-java/branch/setup-ci-build

It comments on the PR requests when the code coverage goes down for example as seen here: #33